packages/Modal/stories/Modal.stories.js

```diff
@@ -140,40 +144,72 @@ storiesOf(`${storyName}/Examples`, module)
             }
           `}
         >
-          <Modal.Header refHeading={refHeading}>Header</Modal.Header>
-          <Modal.Content>
+          <Header refHeading={refHeading}>Header</Header>
+          <Content>
             <p>
               Unicorn next level readymade polaroid, locavore hot chicken forage ennui crucifix tote bag yuccie. Raw
               denim tumblr echo park bushwick hoodie iceland cloud bread iPhone kombucha shoreditch taiyaki woke. Brunch
               ramps cred polaroid, vinyl skateboard portland typewriter jean shorts single-origin coffee flexitarian
               drinking vinegar.
             </p>
-          </Modal.Content>
+          </Content>
         </Modal>
       </Story>
     );
-  });
-
-storiesOf(`${storyName}/Backyard/Sandbox`, module)
-  .add("with form render", () => (
-    <Modal
-      isOpen
-      as="form"
```

Comment:
Removed this story as I don't think we still want to render `as="form"` but prefer to now pass `form` element as a child. Added that story below